scss/_patterns_forms.scss

```diff
@@ -31,20 +31,22 @@
       align-items: baseline;
       display: inline-flex;
       flex-direction: row;
+      flex-wrap: wrap;
+      padding-left: $sph-inner--large;
 
       > * {
-        margin: 0;
+        margin: 0 #{2 * $sph-inner--large} 0 #{-$sph-inner--large};
```

Comment:
I may not fully understand the context here. Why are we adding a padding-left to the form, just to remove it with negative margin on the child elements?
It seems to work the same with them being set to 0.

Reply:
Good point!